### PR TITLE
bugfix/handle-legistar-site-down

### PR DESCRIPTION
**Pull request by dphoria**
### Link to Relevant Issue

This pull request resolves https://github.com/CouncilDataProject/pittsburgh-pa/actions/runs/4393000564/jobs/7693148694

### Description of Changes

The Legistar web site itself for [Pittsburgh, PA is down](https://pittsburgh.legistar.com/). So obviously, the scraper is unable to scrape video URLs.

This PR allows the Legistar scraper to recognize it as web site access error.

### CODE REVIEW

**Comment by dphoria** [comment]
Tests are expected to fail. #132 should be merged first.

File: cdp_scrapers/legistar_utils.py [hunk 1]
```diff
@@ -488,6 +488,9 @@ def get_legistar_content_uris(client: str, legistar_ev: dict) -> ContentUriScrap
         # that is a summary-like page for a meeting
         with urlopen(legistar_ev[LEGISTAR_EV_SITE_URL]) as resp:
             soup = BeautifulSoup(resp.read(), "html.parser")
+            if "server error" in soup.text.lower():
```
Comment (dphoria) [inline cdp_scrapers/legistar_utils.py:491]: We may consider making this parametrized. For now this is the first case of something like this; HTTP 200 returned but is in fact not accessible. If and when we encounter more cases, we can revisit.

**Comment by dphoria** [comment]
This means that for now, we will not be scraping anything from Pittsburgh. Again, the entire https://pittsburgh.legistar.com/ is down :sweat_smile: .

File: cdp_scrapers/legistar_utils.py (hunk 1 above)
Comment (evamaxfield) [reply cdp_scrapers/legistar_utils.py:491]: Oh actually I thought it was good that it started error -- it alerts us it is down. Just that it wasn't clear that is what was happening. Can you raise this as an error but with the context of "Legistar appears to be down" or something similar.

Comment (dphoria) [reply cdp_scrapers/legistar_utils.py:491]: That's a good point. 💡 Agreed.

**Comment by dphoria** [comment]
Take look at this/these failures
https://github.com/CouncilDataProject/cdp-scrapers/actions/runs/4400064672/jobs/7705013373#step:6:2251
These are not 'reset by peer', the links are valid (can be opened on browser). Doesn't seem to be a flaky test. TBD.

If fixes are simple enough, might consider including them in this PR. If the fix ends up being non-trivial, create a separate PR.

**Comment by dphoria** [comment]
Unfortunately(?) [pittsburgh.legistar.com](https://pittsburgh.legistar.com/Calendar.aspx) is no longer down :sweat_smile:

**Comment by dphoria** [comment]
Hm something is odd about the coverage report. Literally no line is hit?... TBD...
https://app.codecov.io/gh/CouncilDataProject/cdp-scrapers/pull/133/blob/cdp_scrapers/legistar_utils.py

**Comment by dphoria** [comment]
> Hm something is odd about the coverage report. Literally no line is hit?... TBD... https://app.codecov.io/gh/CouncilDataProject/cdp-scrapers/pull/133/blob/cdp_scrapers/legistar_utils.py

Looks like it should be fine once #127 is merged then is pulled into this; coverage for that PR is reasonable/sensical.

**Comment by evamaxfield** [comment]
Sounds good! Merge whenever!

**Comment by dphoria** [comment]
I just can't completely trust codecov saying that this repo's test covers less than 10% of the code. I'll investigate/work on that separately. I included some sphinx fixes so I do want to get this merged in.

**Pull request merged**